index.html

```diff
@@ -30,16 +47,16 @@ <h2>The Past</h2>
             <h2>Why Did It Have To Be Boxes...</h2>
             
```

Comment:
Nice job having 2 classes to work with the boxes.  You can do it this way, use an id or you can skip having multiple classes and use the :nth-child() selector.